File: programme/sum-of-numbers/sum-of-numbers.py
#Write a programme to find the sum of n numbers

def sum_of_number(string):
    numbers=list(string.split(" "))
    
    sum=0
    
    for number in numbers:
        sum+=int(number)

    return sum

string="1 2 3 4"
print(sum_of_number(string))

File: programme/sum-of-numbers/sum-of-numbers.java
/* Program to Add N numbers from user and after calculate and display the sum*/

import java.util.*;
class SumOfNumbers
{
public static void main()
{
Scanner inp=new Scanner(System.in);
System.out.print("\n Enter Number of Numbers to be Calculated: ");
int n=inp.nextInt();
int i,sum=0,z;

for(i=0;i<n;i++)   //Entering N numbers
{
System.out.print("\n Enter: ");
z=inp.nextInt();
sum=sum+z;       //Cumulative Sum
}

System.out.println("Sum of the numbers: "+sum);

}
}
